pep-9999.rst

```diff
@@ -0,0 +1,506 @@
+PEP: 9999
+Title: Dependency specification in pyproject.toml using an exploded TOML table
+Author: Laurie Opperman <laurie_opperman@hotmail.com>,
+        Arun Babu Neelicattu <arun.neelicattu@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 02-09-2020
+Post-History:
+
+
+Abstract
+========
+
+This PEP specifies how to write a project's dependencies in a
+``pyproject.toml`` file for packaging-related tools to consume using the fields
+defined in :pep:`621`, as an alternative to the :pep:`508`-based approach
+defined in :pep:`631`. Specifically, the tables ``project.dependencies`` and
+``project.optional-dependencies`` are defined.
+
+
+Motivation
+==========
+
+There are multiple benefits to using TOML tables and other data-types to
+represent requirements rather then :pep:`508` strings:
+
+- Easy initial validation via the TOML syntax.
+
+- Easy secondary validation using a schema, for example a `JSON Schema`_.
+
+- Potential for users to guess the keys of given features, rather than
+  memorising a syntax.
+
+- Users of multiple other popular languages may already be familiar with the
+  TOML syntax.
+
+- TOML directly represents the same data structures as in JSON, and therefore a
+  sub-set of Python literals, so users can understand the hierarchy and type of
+  value
```

Comment:
```suggestion
- Easy initial validation via the TOML syntax.

- Easy secondary validation using a schema, for example a `JSON Schema`_.

- Potential for users to guess the keys of given features, rather than
  memorising a syntax.

- Users of multiple other popular languages may already be familiar with the
  TOML syntax.

- TOML directly represents the same data structures as in JSON, and therefore a
  sub-set of Python literals, so users can understand the hierarchy and type of
  value
```

Does this render appropriately with the newlines between the bullet points?

Reply:
`make pep-9999.html` doesn't show any issues